digiwf-integrations/digiwf-alw-integration/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.muenchendigital.digiwf</groupId>
+        <artifactId>digiwf-integrations</artifactId>
+        <version>0.13.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>digiwf-alw-integration</artifactId>
+    <name>digiwf-alw-integration</name>
+    <packaging>pom</packaging>
+    <description>Integration of the ALW-System used by DigiWF</description>
+
+    <properties>
+        <digiwf-spring-cloudstream-utils.version>0.0.1</digiwf-spring-cloudstream-utils.version>
+    </properties>
```

Comment:
```suggestion

```
The spring-cloudstream-utils are already included in the monorepo. No need to define the version here